credsweeper/filters/group/pem_pattern.py

```diff
@@ -8,5 +8,5 @@ class PEMPattern(Group):
     """PEMPattern"""
 
     def __init__(self, config: Config) -> None:
-        super().__init__(config, GroupType.PATTERN)
+        super().__init__(config, GroupType.DEFAULT)
```

Comment:
Why PEMPattern class is `DEFAULT` group?
It is `PATTERN` group.. isn't it?

Reply:
as i saw, there should be third type. But the item is absent in enum GroupType. Moreover,  the value is not user in future.
So, to reduce extra self.filters assignments - I added DEFAULT
```
class Group(ABC):
    """Abstract Group class"""

    def __init__(self, config: Config, rule_type: GroupType) -> None:
        if rule_type == GroupType.KEYWORD:
            self.filters: List[Filter] = self.get_keyword_base_filters(config)
        elif rule_type == GroupType.PATTERN:
            self.filters: List[Filter] = self.get_pattern_base_filters(config)
        else:
            self.filters: List[Filter] = []
```